packages/core/src/DotLoader/DotLoader.spec.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { render, screen } from 'testing-library'
+import DotLoader from './DotLoader'
+
+describe('DotLoader', () => {
+  it('renders a dot loader', () => {
+    render(<DotLoader data-testid='dot-loader' />)
+
+    const container = screen.getByTestId('dot-loader')
+    const children = container.children
+
+    expect(children).toHaveLength(3)
+    expect(children[0]).toHaveAttribute('color', 'primary')
+    expect(children[0]).toHaveStyleRule('animation', 'bqqVik 2s ease infinite')
+    expect(children[0]).toHaveStyleRule('animation-delay', '0s')
+
+    expect(children[1]).toHaveAttribute('color', 'primary')
+    expect(children[1]).toHaveStyleRule('animation', 'bqqVik 2s ease infinite')
```

Comment:
I think we should consider making this a bit more permissive. I'm concerned that the hash generated for the animation might change over time and cause false negatives.

Reply:
I was unable to fix this since the animation is on the same css line... leaving it as it is unless somebody has a solution